lib/byron/src/Cardano/Wallet/Byron/Compatibility.hs

```diff
@@ -248,10 +248,22 @@ toEpochSlots :: W.EpochLength -> EpochSlots
 toEpochSlots =
     EpochSlots . fromIntegral . W.unEpochLength
 
-toPoint :: W.EpochLength -> W.BlockHeader -> Point ByronBlock
-toPoint epLength (W.BlockHeader sid _ h _)
-    | sid == W.SlotId 0 0 = genesisPoint
-    | otherwise = O.Point $ Point.block (toSlotNo epLength sid) (toByronHash h)
+-- | Magic value for the absence of a block.
+hashOfNoParent :: W.Hash "BlockHeader"
+hashOfNoParent = W.Hash . BS.pack $ replicate 0 32
+
+toPoint
+    :: W.Hash "Genesis"
+    -> W.EpochLength
+    -> W.BlockHeader
+    -> Point ByronBlock
+toPoint genesisH epLength (W.BlockHeader sid _ h _)
+  | h == (conv genesisH) = O.GenesisPoint
+  | otherwise =
+    O.Point $ Point.block (toSlotNo epLength sid) (toByronHash h)
```

Comment:
The indentation is weird here. Indentation should enforce a hierarchy in the information. Here `otherwise` is at the same level as `O.Point`, so the eye treat them similarly: as another branch / guard of the function definition. Indenting it properly makes it easier to process:

```hs
  | h == (conv genesisH) = O.GenesisPoint
  | otherwise =
    O.Point $ Point.block (toSlotNo epLength sid) (toByronHash h)
```

vs

```hs
  | h == (conv genesisH) = O.GenesisPoint
  | otherwise =
      O.Point $ Point.block (toSlotNo epLength sid) (toByronHash h)
```

Reply:
Turns out the newline wasn't needed anyway.